### DISCUSSION TITLE
This was my gateway drug

### DISCUSSION THREAD

**Discussion by embraceentropy** [Jyers/Marlin#789] [General Discussion]
TL:DR - THANK YOU!!! Jyers & this support community are amazing!

My dog is deathly afraid of beeps. I had this (first printer) for one week and was about one day from taking the printer apart to find and disable the speaker all-together, else the printer would've been returned to the store. I found your firmware while searching for alternative methods. Now, I can't imagine where I would be without it. I've gone from being intimidated flashing pre-compiled FW to now experimenting with different parameters, compiling, and flashing over and over again. It also gave me the confidence to tweak, adjust, & calibrate. Now my prints get better every day!

Whether you know it or not...you're becoming a gateway to make beginners into power users. The discussion section of this git alone is amazing with many contributors who make this hobby/trade accessible.
